GenderPayGap.WebUI/Controllers/ApiForTomsAppController.cs

```diff
@@ -12,13 +12,13 @@
 
 namespace GenderPayGap.WebUI.Controllers
 {
-    [Route("api")]
-    public class ApiController : Controller
+    [Route("api/toms-app")]

```

Comment:
should we be using the "official" name, the Reports Reader?

Reply:
Yeh that makes sense